src/css/profile/mobile/common/calendar.less

```diff
@@ -0,0 +1,119 @@
+body {
+	margin: 0;
+}
+.calendarView {
+	width: 100%;
+	border-spacing: 0;
+}
+tr {
+	td {
+		height: 32 * @px_base;
+		text-align: center;
+		vertical-align: middle;
+		font-size: 15 * @px_base;
+		.font(regular); 
+		color: #454545;
+		padding: 0;
+		&:only-child {
+			width: 100%;
+		}
+		&:nth-last-child(2) {
+			width: 50%;
+		}
+		&:nth-last-child(2)+td {
+			width: 50%;
+		}
+		&:nth-last-child(3) {
+			width: 33.3%;
+		}
+		&:nth-last-child(3)~td {
+			width: 33.3%;
+		}
+		&:nth-last-child(4) {
+			width: 25%;
+		}
+		&:nth-last-child(4)~td {
+			width: 25%;
+		}
+		&:nth-last-child(5) {
+			width: 20%;
+		}
+		&:nth-last-child(5)~td {
+			width: 20%;
+		}
+		&:nth-last-child(6) {
+			width: 16.6%;
+		}
+		&:nth-last-child(6)~td {
+			width: 16.6%;
+		}
+		&:nth-last-child(7) {
+			width: 14.2%;
+		}
+		&:nth-last-child(7)~td {
+			width: 14.2%;
+		}
+		&:nth-child(7) {
+			color: #c95151;
```

Comment:
How about the define the variable for the calendar color to the "theme.color.less"?